### PR TITLE
Fix #331: Correct a few typos

### LINKED ISSUE #331: Typos in the Guidelines
There are multiple typos in the guidelines, for example:

* `IDs must opaque strings` should be `IDs must be opaque strings`
* `different versionsof a resource` should be `different versions of a resource`
* `make sure that all constraints a clearly defined` should be `make sure that all constraints are clearly defined`

etc.

It would be nice to proof-read and correct these typos as they may sometimes distract a bit from the main points made in the document.

### PR DESCRIPTION
**Pull request by antivanov**
* Correct a few typos
 * Interchange the terms `page` and `offset` in *"137. Use Conventional Query Strings"*

### CODE REVIEW

**Comment by hjacobs** [comment]
Thank you! :smile:

**Comment by antivanov** [comment]
Please, check that the small corrections make sense and let me know if PR needs to be updated. Would be nice to have a couple of reviewers looking over the corrections 😄

File: chapters/naming.adoc [hunk 1]
```diff
@@ -83,10 +83,10 @@ should have an entity specific alias, like sku
 below. Hint: You can use size as an alternate query string.
 * `cursor` — key-based page start. See <<pagination>> section below.
 * `offset` — numeric offset page start. See <<pagination>> section below.
-Hint: In combination with limit, you can use page as an alternative to
-offset.
+Hint: In combination with limit, you can use offset as an alternative to
```
Comment (tkrop) [inline chapters/naming.adoc]: I think, this change is not what was intended by the hint. Our standard for pagination is `offset`/`cursor` and `limit`. The hint was supposed to add `page` as an alternative to `offset`.

As far as I can see pagination is not suggesting this replacement, thus I would suggest to drop this hint here, too.

Comment (antivanov) [reply chapters/naming.adoc]: Thanks, looks like I misunderstood how the pagination is defined. So `page` in the sentence 

>Hint: In combination with `limit`, you can use `page` as an alternative to
offset.

has the same meaning as `cursor` when slightly rephrased:

>Hint:  As pagination is defined by parameters `offset|cursor` and `limit`, in combination with `limit`, you can use `cursor` as an alternative to `offset`.

Will revert this change to use the original text.

Comment (antivanov) [reply chapters/naming.adoc]: Reverted the change in the hint.

**Comment by antivanov** [comment]
Thanks for reviewing, reverted the hint for *"Use Conventional Query Strings [137]"* back to the original.

**Comment by tkrop** [comment]
👍

**Comment by maxim-tschumak** [comment]
:+1:

**Comment by tkrop** [comment]
👍

**Pull request merged**